samza-core/src/main/scala/org/apache/samza/diagnostics/DiagnosticsStreamMessage.java

```diff
@@ -185,6 +203,14 @@ public Integer getNumStoresWithChangelog() {
         CONTAINER_NUM_STORES_WITH_CHANGELOG_METRIC_NAME);
   }
 
+  public Integer getContainerThreadPoolSize() {
+    return (Integer) getFromMetricsMessage(GROUP_NAME_FOR_DIAGNOSTICS_MANAGER, CONTAINER_THREAD_POOL_SIZE_METRIC_NAME);
+  }
+
+  public Long getMaxHeapSize() {
```

Comment:
Minor: can you keep the same ordering for methods? So could you move this one above `getContainerThreadPoolSize`